.github/workflows/deploy-production.yaml

```diff
@@ -0,0 +1,37 @@
+name: "Deploy Production"
+on:
+  workflow_dispatch: {}
+
+env:
+  REGISTRY: ghcr.io
+
+jobs:
+  deployment:
+    runs-on: 'ubuntu-latest'
+    container: dtzar/helm-kubectl:3.9.4
+    env:
+      DEPLOY_IMAGE: ghcr.io/${{ github.repository }}
+      WORKER_IMAGE: ghcr.io/${{ github.repository }}/worker
+      HELM_EXPERIMENTAL_OCI: 1
+      HELM_RELEASE_NAME: gbh-production
```

Comment:
they don't _have_ to but we are only deploying this one app to the namespace, so they may as well.

Reply:
👍

Reply:
interpret_visibility_actor and child_relationships_job. We use this decorator pattern whenever we override a file from a gem dependency. This makes it easier to target the exact change we want instead of copying the whole file over.